core/setup.py

```diff
@@ -61,7 +61,8 @@ def create_version_file():
     'pandas>=1.0.0,<2.0',
     'psutil>=5.0.0,<=5.7.0',  # TODO: psutil 5.7.1/5.7.2 has non-deterministic error on CI doc build -  ImportError: cannot import name '_psutil_linux' from 'psutil'
     'scikit-learn>=0.22.0,<0.24',
-    'networkx>=2.3,<3.0'
+    'networkx>=2.3,<3.0',
+    'autograd'
```

Comment:
do you suspect there may be minimum required version of autograd?

Reply:
Good point, the current one is 1.3, I'll add >=1.3 to make sure. We need >= 1.2 for sure, when they changed the API.